.changeset/purple-ideas-switch.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/shared-object-base": minor
+---
+---
+"section": deprecation
+---
+
+The function `processCore` on `SharedObject` and `SharedObjectCore` is now deprecated.
+
+A new function `processMessagesCore` has been added in its place which will be called to process multiple messages instead of a single one on the channel. This is part of a feature called "Op bunching" where contiguous ops in a grouped batch are bunched and sent together to the shared object for processing.
```

Comment:
```suggestion
A new function `processMessagesCore` has been added in its place which will be called to process multiple messages instead of a single one on the channel. This is part of a feature called "Op bunching" where contiguous ops in a grouped batch are bunched and ingested together by shared object for processing.
```
I'm not married to the phrasing I'm suggesting. I mostly want to avoid the word "sent" because I fear it might cause confusion since "sending" is generally used for over-the-network exchanges.